include/SFML/System/Time.hpp

```diff
@@ -49,6 +50,13 @@ class Time
     ////////////////////////////////////////////////////////////
     constexpr Time();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Construct from std::chrono::duration
+    ///
+    ////////////////////////////////////////////////////////////
+    template<class Rep, class Period>
+    constexpr Time(const std::chrono::duration<Rep, Period>& duration);
```

Comment:
Is that constructor not explicit on purpose?

Reply:
That's right. We want to allow these implicit conversions.